### PR TITLE
updated readme and added environmentl.yml for conda build.

### LINKED ISSUE #13: Update Build Docs and Anaconda Environment

### PR DESCRIPTION
**Pull request by gsn9**
Closes #13

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -3,8 +3,14 @@
 Repository for the CartoCosmos Capstone Team.
 
 ### Building and running on localhost
+#
+
+Install conda:
+* [Windows](https://docs.conda.io/projects/conda/en/latest/user-guide/install/windows.html) 
+* [Mac](https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html)
+* [Linux](https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html)
+
```
Comment (amystamile) [inline app/README.md:12]: Should we have instructions like `conda env create -n somename -f environment.yml`?

File: app/environment.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - ca-certificates=2021.10.8=ha878542_0
+  - gh=2.5.0=ha8f183a_0
+  - icu=68.1=h2531618_0
+  - libgcc-ng=9.3.0=h5101ec6_17
+  - libgomp=9.3.0=h5101ec6_17
+  - libstdcxx-ng=9.3.0=hd4cf53a_17
+  - libuv=1.40.0=h7b6447c_0
+  - nodejs=16.13.1=hb931c9a_0
+  - openssl=1.1.1m=h7f8727e_0
+  - zlib=1.2.11=h7f8727e_4
```
Comment (amystamile) [inline app/environment.yml:17]: I was expecting something super simple like:

dependencies:
    - nodejs

I don't think we need all the lib stuff isolated in an evironment

File: app/environment.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+prefix: /home/gavin/miniconda
```
Comment (amystamile) [inline app/environment.yml:18]: This wouldn't work for everyone else.

File: app/README.md (hunk 1 above)
Comment (gsn9) [reply app/README.md:12]: Yeah I'll add that in when I clean this pr up

File: app/environment.yml (hunk 3 above)
Comment (gsn9) [reply app/environment.yml:18]: I'll get rid of it.

**Pull request closed without merging**